client/data/disputes/resolvers.js

```diff
@@ -38,7 +38,7 @@ const formatQueryFilters = ( query ) => ( {
 			: query.search,
 	status_is: query.statusIs,
 	status_is_not: query.statusIsNot,
-	locale: query.userLocale,
+	locale: query.locale,
 } );
```

Comment:
Similar comment to above: are these intentionally allow-lists or could we simplify by using generic processing for query args.